app/client/src/app/components/form-components/autoCompleteWrapper.js

```diff
@@ -2,6 +2,7 @@ import React, { useState } from "react";
 
 import Autosuggest from "react-autosuggest";
 
+import alternativeSchoolNames from "./alternativeSchoolNames.json";
```

Comment:
Not sure if this is the "right" way to pull in that alternativeSchoolNames data. Open to other suggestions!

Reply:
lgtm!